mysql/assets/recommended_monitors/mysql_replication_seconds_behind_master.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "[MySQL] Replication seconds behind master",
+    "type": "query alert",
+    "query": "avg(last_5m):max:mysql.replication.seconds_behind_master{*} by {host} > 3000",
```

Comment:
Why is the threshold 3000? If it's in seconds it means 50min.
If you have a 10 seconds lag it would alreaby causing issues.

Reply:
will change the threshold to something substantially shorter.